generation/Windows/um/Presentation/generate.rsp

```diff
@@ -0,0 +1,21 @@
+@../../../settings.rsp
+@../../../remap.rsp
+--file
+um-Presentation.h
+--methodClassName
+Windows
+--namespace
+TerraFX.Interop.Windows
+--output
+../../../../sources/Interop/Windows/Windows/um/Presentation
+--test-output
+../../../../tests/Interop/Windows/Windows/um/Presentation
+--traverse
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.26100.0/um/Presentation.h
+--with-librarypath
+CreatePresentationFactory=dcomp
+--with-using
+CompositionFrameDisplayInstance=TerraFX.Interop.DirectX
+ICompositionFramePresentStatistics=TerraFX.Interop.DirectX
+IPresentationSurface=TerraFX.Interop.DirectX
+IPresentationSurface2=TerraFX.Interop.DirectX
```

Comment:
You should be able to get these automatic by adding the type it uses to https://github.com/terrafx/terrafx.interop.windows/blob/main/generation/remap-namespaces.rsp, which should just tell the system that `DXGI_COLOR_SPACE_TYPE` is in `TerraFX.Interop.DirectX`